logistics_project/apps/malawi/extensions/rapidsms/contact.py

```diff
@@ -44,4 +47,15 @@ def hsa_id(self):
             else:
                 return ""
         else:
-            return ""
\ No newline at end of file
+            return ""
+
+    def translate(self, message):
```

Comment:
note that I don't *think* this project is used in any language besides english (but I might be wrong about that)